Apply coding standards

alvinjohnsonso opened the pull request:
- Fixed spacings and indentations.
- Added GPL License file.
- Updated README.md file to comply with the zencart requirements.
- Added build script for generating the zip file.

alvinjohnsonso committed:
fixed spacings and indentation to comply with coding standards

alvinjohnsonso committed:
added gpl license file

alvinjohnsonso committed:
Updated README file with new instructions on installation and removing the module

alvinjohnsonso committed:
added new lines at eof

alvinjohnsonso committed:
added build script

alvinjohnsonso committed:
fixed markdown format for noted instruction

alvinjohnsonso committed:
modified noted instructions

alvinjohnsonso committed:
added FAQ section to readme file

alvinjohnsonso committed:
transferred set and remove widget logic to another php file

alvinjohnsonso commented on YOUR_ADMIN_FOLDER/widget.php:
```diff
@@ -0,0 +1,51 @@
+<?php
```
The code here is just gotten from YOUR_ADMIN_FOLDER/tawk_to_widget.php. Transferred it here since there were problems encountered for ZenCart 1.5.7 for having the logic in the same file.

GeekOfAges replied:
`widget` is far too generic a name and has high chance of conflicts. Perhaps something like `tawk_widget_manager.php` to avoid chances of conflicts

GeekOfAges commented on YOUR_ADMIN_FOLDER/widget.php:
```diff
@@ -0,0 +1,51 @@
+
+require('includes/application_top.php');
+
+if (isset($_GET['set'])) {
```
Instead of using `isset($_GET['set'])`, providing `cmd` or action param and then selecting from 2 functions to be executed is the way to go here IMHO

alvinjohnsonso committed:
renamed tawk_to_widget.php to tawk_to_widget_admin.php in YOUR_ADMIN_FOLDER;
renamed widget.php to tawk_to_widget_manager.php in YOUR_ADMIN_FOLDER;
removed unnecessary die clause;
updated set and remove widget request handler, added request method validator and changed query param to actionType;

alvinjohnsonso committed:
removed unused global variable

alvinjohnsonso committed:
version bumped to 1.1.0
